samples/dashboards/demo/accounting/demo.js

```diff
@@ -0,0 +1,637 @@
+    [Date.UTC(2021, 8), 8500000, 7000000, 9700000, null, null, null, null],
+    [Date.UTC(2021, 9), 8000000, 8500000, 9200000, null, null, null, null],
+    [Date.UTC(2021, 10), 8500000, 8000000, 9700000, null, null, null, null],
+    [Date.UTC(2021, 11), 8000000, 8300000, 9200000, null, null, null, null],
+    [Date.UTC(2022, 0), 8500000, 8600000, 9700000, null, null, null, null],
+    [Date.UTC(2022, 1), 8000000, 8800000, 9200000, null, null, null, null],
+    [Date.UTC(2022, 2), 8500000, 8500000, 9700000, null, null, null, null],
+    [Date.UTC(2022, 3), 8000000, 7500000, 9200000, null, null, null, null],
+    [Date.UTC(2022, 4), 8500000, 8700000, 9700000, null, null, null, null],
+    [Date.UTC(2022, 5), 8000000, 9500000, 10200000, null, null, null, null],
+    [Date.UTC(2022, 6), 8500000, 8200000, 8700000, null, null, null, null],
+    [Date.UTC(2022, 7), 8000000, 8900000, 10700000, null, null, null, null],
+    [Date.UTC(2022, 8), 8500000, 9200000, 9200000, null, null, null, null],
+    [Date.UTC(2022, 9), 8000000, 7800000, 7800000, null, null, null, null],
+    [Date.UTC(2022, 10), 8500000, 8500000, 8000000, null, null, null, null],
+    [Date.UTC(2022, 11), 8000000, 7800000, 8000000, null, null, null, null],
+    [Date.UTC(2023, 0), 8500000, 6900000, 8700000, null, null, null, null],
+    [Date.UTC(2023, 1), 8000000, 7100000, 8900000, null, null, null, null],
+    [Date.UTC(2023, 2), 8500000, 7200000, 8000000, null, null, null, null],
+    [Date.UTC(2023, 3), 8000000, 7400000, 8000000, null, null, null, null],
+    [Date.UTC(2023, 4), 8500000, 6900000, 8700000, null, null, null, null],
+    [Date.UTC(2023, 5), 8000000, 7100000, 8900000, null, null, null, null],
+    [Date.UTC(2023, 6), 8500000, 7500000, 8000000, null, null, null, null],
+    [Date.UTC(2023, 7), 8000000, 7300000, 8700000, null, null, null, null],
+    [Date.UTC(2023, 8), 8500000, 7600000, 8900000, null, null, null, null],
+    [Date.UTC(2023, 9), 8000000, 7600000, 8900000, null, null, null, null],
+    [Date.UTC(2023, 10), 7500000, null, null, 79e5, 82e5, 69e5, 90e5],
+    [Date.UTC(2023, 11), 7600000, null, null, 80e5, 83e5, 70e5, 92e5],
+    [Date.UTC(2024, 0), null, null, null, 79e5, 83e5, 70e5, 91e5],
+    [Date.UTC(2024, 1), null, null, null, 80e5, 80e5, 74e5, 92e5],
+    [Date.UTC(2024, 2), null, null, null, 82e5, 81e5, 73e5, 94e5],
+    [Date.UTC(2024, 3), null, null, null, 81e5, 84e5, 74e5, 92e5],
+    [Date.UTC(2024, 4), null, null, null, 79e5, 82e5, 72e5, 92e5],
+    [Date.UTC(2024, 5), null, null, null, 81e5, 82e5, 73e5, 90e5],
+    [Date.UTC(2024, 6), null, null, null, 82e5, 83e5, 74e5, 91e5],
+    [Date.UTC(2024, 7), null, null, null, 84e5, 85e5, 75e5, 92e5],
+    [Date.UTC(2024, 8), null, null, null, 83e5, 85e5, 74e5, 93e5],
+    [Date.UTC(2024, 9), null, null, null, 82e5, 82e5, 75e5, 93e5],
+    [Date.UTC(2024, 10), null, null, null, 83e5, 83e5, 74e5, 92e5],
+    [Date.UTC(2024, 11), null, null, null, 81e5, 84e5, 75e5, 93e5]
+];
```

Comment:
We can push data to the public HC server, like in AWS. We will minify the demo then.

You can discuss it with @goransle

Reply:
I'm just wondering if it would be enough to change the data format to csv and transfer it to HTML. `demo.js` will then be clean. Unless it's better to host it?